platform/src/main/java/org/stellar/anchor/platform/service/TransactionService.java

```diff
@@ -192,6 +193,11 @@ private GetTransactionResponse patchTransaction(PatchTransactionRequest patch)
     return toGetTransactionResponse(txn, assetService);
   }
 
+  public CreateTransactionPaymentResponse createCustodyTransactionPayment(
```

Comment:
I don't think that we need this method. We can simply call `custodyTransactionService.createCustodyTransactionPayment(txnId, requestBody)` from controller